gulpfile.babel.js

```diff
@@ -69,13 +72,13 @@ function lint(files, options) {
 gulp.task('lint:client', lint(`${clientDir}/**/*.jsx`));
 gulp.task('lint:server', lint(`./${serverDir}/server.js`));
 gulp.task('lint:step-templates', () => {
-  return gulp.src('./step-templates/*')
+  return gulp.src('./step-templates/*.json')
```

Comment:
Ok, we need it because we have logos folder there. I added a test that will report any files without proper file extension.